test/functionalTest/cases/3068_non_legacy_notifications/non_legacy_notifications_with_accumulator.test

```diff
@@ -0,0 +1,113 @@
+# Copyright 2019 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Simple forwarded query in NGSIv2
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 38,186-187,235 IPV4
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. Register entities of type T, with ID .*, and attribute A1, for the accumulator
+# 02. Query entities of type T - get empty response as the accumulator doesn't respond
+# 03. Dump the accumulator, see the forwarded query
+#
+
+echo "01. Register entities of type T, with ID .*, and attribute A1, for the accumulator"
+echo "=================================================================================="
+payload='{
+  "dataProvided": {
+    "entities": [
+      {
+        "type": "T",
+        "idPattern": ".*"
+      }
+    ],
+    "attrs": [ "A1" ]
+  },
+  "provider": {
+    "http": {
+      "url": "http://localhost:'${LISTENER_PORT}'/v2"
+    },
+    "legacyForwarding": false
```

Comment:
Do we have some test case testing the ommision of the legacyForwarding field (as equivalente to "V2") ?

Reply:
No test, and I believe I implemented that as equivalent to V1 ...
No wait, that was the famous "JSON" that in some cases is empty. If empty =>V1.
V2 only for "normalized" etc.
 
Adding a test to make sure how it works. It's a bit confusing ...

Reply:
Not confusing :), the rule is:

legacyForwarding ommission == legacyForwarding: false

But we should have a test to cover it.

Reply:
Did this .test finally added?

Reply:
Here: 3068_non_legacy_notifications/legacy_forwarding_true_false_default_with_accumulator.test

Reply:
I found it:

```
 # 01. Registration of E1 without Legacy Forwarding, for the accumulator - forward to be in NGSIv2 format
```

NTC